src/System.IO.FileSystem/src/project.json

```diff
@@ -23,14 +23,14 @@
         "dotnet5.4"
```

Comment:
Any reason you switch ordering for 1.3 and 1.7? If you keep them reordered then you need to change the imports to match as well.